docs/CITA Release flow.md

```diff
@@ -0,0 +1,202 @@
+# CITA Release flow
+
+## 1. 发版计划
+
+* 确定发版时间
+* 确定 Release Master （整个发版过程由 Release Master 负责）
+* 确定发版测试负责人
+
+## 2. 冻结版本
+* 冻结代码至新建的 Release 分支，需提前一周以上冻结
+* 冻结分支应该有最新的版本号等，需为可发布版本如无BUG
+
+### 准备分支
+
+#### 版本的类型
+
+使用语义化版本，遵循 [Semantic Versioning]
+
+摘录部分如下：
+
+> 版本格式：主版本号.次版本号.修订号，版本号递增规则如下：
+>
+> 1. 主版本号：当你做了不兼容的 API 修改，
+> 2. 次版本号：当你做了向下兼容的功能性新增，
+> 3. 修订号：当你做了向下兼容的问题修正。
+> 先行版本号及版本编译元数据可以加到“主版本号.次版本号.修订号”的后面，作为延伸。
+
+#### 创建发布分支
+
+***新分支命名遵循 `release-*` 格式***
+
+首先获取最新的代码，具体操作如下：
+
+```shell
+git fetch origin
+```
+
+其中 `origin` 代表 `CITA` 官方的代码仓库地址，可通过 `git remote -v` 查看。
+
+流程分为版本发布（主版本号 x 及次版本号 y）以及修订补丁发布（修订号 z）。
+
+##### 版本发布
+
+从最新 `develop` 创建新的分支 `release-x.y.0`，具体操作如下：
+
+```shell
+git checkout origin/develop -b release-x.y.0
+```
+
+##### 修订补丁发布
+
+从需要修订版本的 `tag`(`vx.y.0`) 创建新的分支 `release-x.y.1`，具体操作如下：
+
+```shell
+git checkout tags/vx.y.0 -b release-x.y.1
```

Comment:
is it correct?
I think it should be `git checkout -b <New Branch Name> <TAG Name>`

Reply:
fixed